packages/gensx-openai/__mocks__/openai.ts

```diff
@@ -81,6 +81,14 @@ const mockCreateMethod = vi
     }
     // Handle tool calls
     else if (params.tools && params.tools.length > 0) {
+      // Check if there's an output_schema tool in the tools array
+      const hasOutputSchemaTool = params.tools.some(
+        (tool) =>
+          typeof tool === "object" &&
+          "function" in tool &&
+          tool.function.name === "output_schema",
```

Comment:
Should we namespace this to reduce the slim chance of collisions?

```suggestion
          tool.function.name === "gensx_output_schema",
```

Reply:
hmm interesting idea. I like it in principle but the model has no concept of what gensx is so i could see this hurting performance. in my mind `output_schema` as the tool name is what makes its purpose super obvious to the model and the description is less important.  

I'm a bit less worried about a name collison here than i am with the `outputSchema` prop we added

Reply:
👍